src/compiler/program.ts

```diff
@@ -10,6 +10,7 @@ module ts {
     /** The version of the TypeScript compiler release */
     export let version = "1.5.0.0";
 
+    /** @param setParentNodes True to skip binding stage setting parent nodes (useful when only interested in syntactic parsing). */

```

Comment:
Thinking about it further and running the idea by @CyrusNajmabadi, perhaps we want to make it default to `true`. That way users won't run into issues where they don't get parent pointers set.